### PR TITLE
Feature/blog Johana

### PR DESCRIPTION
**Pull request by johanaludwig**

### CODE REVIEW

File: staff/johana-ludwig/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Blog de idiomas y viajes</title>
+</head>
+<body>
+    <header>
+        <h1>Viaja y aprende</h1>
+        <section>
```
Comment (robert-z) [inline staff/johana-ludwig/blog/index.html]: In this case, the `<section>` tag is not necessary

File: staff/johana-ludwig/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    </header>
+    <main>
+            <article>
+                <header>
+                    <h2>5 lugares que visitar en Berlín</h2>
+                    <p><small>Autor: <a href="#">J.LO</a></small></p>
+                </header>
+                <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr...<a href="post.html">Ver más</a></p>
+            </article>
+            <br>
```
Comment (robert-z) [inline staff/johana-ludwig/blog/index.html]: `<br>` tag is not necessary. If you want a space between two `<articles>` you must to use CSS styles

Comment (johanaludwig) [reply staff/johana-ludwig/blog/index.html]: ok thank!

File: staff/johana-ludwig/blog/index.html (hunk 1 above)
Comment (johanaludwig) [reply staff/johana-ludwig/blog/index.html]: you are right :)

**Pull request merged**